package.json

```diff
@@ -11,10 +11,11 @@
     "copy-env": "if ! [ -e .env ]; then cp .env.tpl .env; fi;"
   },
   "dependencies": {
+    "@material-ui/core": "^4.2.0",
     "axios": "0.19.0",
     "prop-types": "15.6.2",
-    "react": "16.5.1",
-    "react-dom": "16.5.1",
+    "react": "16.8",
```

Comment:
why you're updating packages in design PR?

Reply:
@material-ui/core@4.2.0 requires react and react-dom ^16.8.0

Reply:
ok

Reply:
I left that as it is